ui/components/app/confirm-page-container/confirm-page-container-content/index.scss

```
@@ -8,6 +8,10 @@
   display: flex;
   flex-direction: column;
 
+  &--with-top-border {
+    border-top: 1px solid var(--geyser);
```

```suggestion
    border-top: 1px solid var(--ui-grey);
```

we can replace `--geyser` since it's a deprecated color (ref: [design-system/deprecated-colors.scss](https://github.com/MetaMask/metamask-extension/blob/develop/ui/css/design-system/deprecated-colors.scss)).  I think `--ui-grey` is the closest match from the new color system. In a follow-up PR, we can replace the remanding `--geyser` occurrences.

convo to deprecate deprecated colors here: https://github.com/MetaMask/metamask-extension/issues/13247

---

@georgewrmarshall

---

Thanks for the tip. This color matches the border from [ConfirmPageContainerSummary](https://github.com/MetaMask/metamask-extension/pull/13314/files#diff-8862eff580778580c03d77a36826a57fb1855947d61a610e53117e0072b900afR66), so I am going to leave it for now since it's not the same exact color, but that's great to know!

---

ah, good reason to keep it for now for consistency 👍